ansible_rulebook/util.py

```diff
@@ -50,7 +50,7 @@
 
 
 def decrypted_context(
-    obj: Union[Dict, List, str, bool, int]
+    obj: Union[Dict, List, str, bool, int],
```

Comment:
@zkayyali812 Is this change needed?

Reply:
This change was done automatically by black